### PR TITLE
Propose new JHipster Vulnerability PR message body

### PR DESCRIPTION
**Pull request by JLLeitschuh**

### CODE REVIEW

File: jhipster_rng_vulnerability/PR_MESSAGE.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+This vulnerability has a CVSS v3.0 Base Score of [9.8/10](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?name=CVE-2019-16303&vector=AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H&version=3.1&source=NIST).
+
+[POC code](https://medium.com/@alex91ar/the-java-soothsayer-a-practical-application-for-insecure-randomness-c67b0cd148cd) has existed since March 3rd, 2018 for taking one RNG value generated by `RandomStringUtils` and reversing it to generate all of the past/future RNG values.
+
+This contribution is a part of a submission to the [GitHub Security Lab](https://securitylab.github.com/) Bug Bounty program.
+
+## Potential Impact Technical
+
+Using one password reset token from your app combined with the POC above, an attacker can determine all future password reset tokens to be generated by this server. This allows an attacker to pick and choose what account they would like to takeover by sending account password reset requests for targeted accounts.
```
Comment (xcorail) [inline jhipster_rng_vulnerability/PR_MESSAGE.md]: I would put this paragraph at the beginning ... perhaps just after `This is a security fix ...`

**Comment by xcorail** [comment]
Hey @anticomputer, mind giving some feedback on this text?
The idea is to maximise the `merge` / `report user` ratio on the PR.

**Comment by anticomputer** [comment]
> Hey @anticomputer, mind giving some feedback on this text?
> The idea is to maximise the `merge` / `report user` ratio on the PR.

Looks good to me.

File: jhipster_rng_vulnerability/PR_MESSAGE.md (hunk 1 above)
Comment (JLLeitschuh) [reply jhipster_rng_vulnerability/PR_MESSAGE.md]: Done!!

File: jhipster_rng_vulnerability/PR_MESSAGE.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+[SECURITY] CVE-2019-16303 - JHipster Vulnerability Fix - Use CSPRNG in RandomUtil
+
+---
+
+This is a security fix for a  vulnerability in your [JHipster](https://www.jhipster.tech/) generated `RandomUtil.java` file(s).
+
+## Technical Impact
+
+Using one password reset token from your app combined with the POC above, an attacker can determine all future password reset tokens to be generated by this server. This allows an attacker to pick and choose what account they would like to takeover by sending account password reset requests for targeted accounts.
```
Comment (xcorail) [inline jhipster_rng_vulnerability/PR_MESSAGE.md]:
```suggestion
Using one password reset token from your app combined with the POC below, an attacker can determine all future password reset tokens to be generated by this server. This allows an attacker to pick and choose what account they would like to takeover by sending account password reset requests for targeted accounts.
```

**Pull request merged**